### PR TITLE
Configuration verb

### PR DESCRIPTION
**Pull request by juliangiebel**
Adds an admin only verb for opening the user interface to:

- ConfigurationComponent (DisposalMailingUnit)
- DisposalTaggerComponent
- DisposalRouterComponent

Any future entities using the Configuration component will also benefit from this.

Tested as admin ghost.

### CODE REVIEW

**Review by ShadowCommander — APPROVED** [review]

File: Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs [hunk 1]
```diff
@@ -133,8 +141,7 @@ void IActivate.Activate(ActivateEventArgs args)
             var activeHandEntity = hands.GetActiveHand?.Owner;
             if (activeHandEntity == null)
             {
-                UpdateUserInterface();
-                UserInterface?.Open(actor.playerSession);
+                OpenUserInterface(actor);
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs:144]: Any way to consolidate this with the above? The code looks the same.

File: Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs [hunk 2]
```diff
@@ -143,5 +150,37 @@ public override void OnRemove()
             base.OnRemove();
             UserInterface?.CloseAll();
         }
+
+        [Verb]
+        public sealed class ConfigureVerb : Verb<DisposalTaggerComponent>
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs:155]: On the tagger it seems to have the same functionality as clicking it as I can't use the verb out of range.

File: Resources/Textures/Interface/VerbIcons/settings.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-settings"><circle cx="12" cy="12" r="3"></circle><path d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"></path></svg>
```
Comment (metalgearsloth) [inline Resources/Textures/Interface/VerbIcons/settings.svg:1]: Is this a new one? Not sure if we dump licences for the svgs anywhere.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs (hunk 2 above)
Comment (juliangiebel) [reply Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs:155]: The verb is mainly for opening the interface as an admin ghost.

File: Resources/Textures/Interface/VerbIcons/settings.svg (hunk 3 above)
Comment (juliangiebel) [reply Resources/Textures/Interface/VerbIcons/settings.svg:1]: The icon is under the MIT License

File: Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs (hunk 1 above)
Comment (juliangiebel) [reply Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs:144]: What should I try to consolidate this line / function with?

File: Resources/Textures/Interface/VerbIcons/settings.svg (hunk 3 above)
Comment (metalgearsloth) [reply Resources/Textures/Interface/VerbIcons/settings.svg:1]: I think you meant the CC one but sounds fine.

File: Content.Server/GameObjects/Components/Disposal/DisposalRouterComponent.cs [hunk 4]
```diff
@@ -95,22 +101,25 @@ private void OnUiReceiveMessage(ServerBoundUserInterfaceMessage obj)
...
             if (!Anchored)
                 return false;
+
+            var groupController = IoCManager.Resolve<IConGroupController>();
             //Check if player can interact in their current state
-            if (!ActionBlockerSystem.CanInteract(playerEntity) || !ActionBlockerSystem.CanUse(playerEntity))
+            if (!groupController.CanAdminMenu(session) && (!ActionBlockerSystem.CanInteract(session.AttachedEntity) || !ActionBlockerSystem.CanUse(session.AttachedEntity)))
                 return false;
 
             return true;
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Disposal/DisposalRouterComponent.cs:119]: This bit (in reference to below).

**Comment by metalgearsloth** [comment]
Dupe code will probably be fixed later given mailing is a copy of the disposal unit.

**Pull request merged**